cockpit/src/subscriptions-client.js

```diff
@@ -130,6 +130,9 @@ function dbus_str(value) {
     };
 }
 
+client.closeRegisterDialog = false;
+client.dfd = cockpit.defer();
```

Comment:
Wouldn't the deferred object get reused then? cockpit's deferred comes from jquery's deferred: https://api.jquery.com/jquery.deferred/ , which really shouldn't be reused (its `resolve` or `reject` should only called once...). I'm not sure what consequence this may have (quick googling suggests subsequent `resolve`/`reject` calls may be ignored).

Reply:
What I'd like to see is either a change to the PR or an explanation in the form of code comments of why this doesn't cause issues (depending on whether there are problems with reusing `client.dfd`).